tests/test_frechet_mean.py

```diff
@@ -59,29 +123,47 @@ def test_estimate_and_belongs_sphere(self):
 
     @geomstats.tests.np_and_pytorch_only
     def test_variance_sphere(self):
+        dim = 5
         point = gs.array([0., 0., 0., 0., 1.])
-        points = gs.zeros((2, point.shape[0]))
+        points = gs.zeros((2, dim))
         points[0, :] = point
         points[1, :] = point
 
         result = variance(
             points, base_point=point, metric=self.sphere.metric)
-        expected = gs.array([[0.]])
+        expected = gs.array(0.)
+
+        self.assertAllClose(expected, result)
+
+    @geomstats.tests.np_and_pytorch_only
+    def test_estimate_default_gradient_descent_sphere(self):
+        dim = 5
+        point = gs.array([0., 0., 0., 0., 1.])
+        points = gs.zeros((2, dim))
+        points[0, :] = point
+        points[1, :] = point
```

Comment:
maybe use `points = gs.array([point, point])` so it doesn't rely on assignement and will work with tf?

Reply:
Good eye! Fixed.